### PR TITLE
Feat/add header navigation menu

### PR DESCRIPTION
**Pull request by RaphaelEscrig**
This PR implements the header navigation. Because all the pages aren't done at the moment, we may redirect to the design. This allow the product / market team to use the portal for their future demos while we develop all the pages.

### CODE REVIEW

File: package.json
Comment (ErikssonJoakim) [inline package.json:30]: This is not in the scope of the PR. We should wait until #74 is merged before rebasing this PR and then we can review

**Comment by fredericvilcot** [comment]
@RaphaelEscrig 
Could please rebase this PR from `main`?
Thx!

**Review by ErikssonJoakim — APPROVED** [review]

File: src/components/layout/Layout.tsx
Comment (fredericvilcot) [inline src/components/layout/Layout.tsx:203]: Why do you use a `uuid` as key?

File: src/i18n/header/header_fr.json
Comment (ccamel) [inline src/i18n/header/header_fr.json:4]:
```suggestion
  "interact": "Interagir",
```

File: src/components/layout/Layout.tsx
Comment (ccamel) [reply src/components/layout/Layout.tsx:203]: Oh! 🧐 Well spotted. Keys must be *stable*. Don't use uuid for that as it assigns new key for each element every time... Since your list is constant, just use the index, it will do the trick.

**Review by ccamel — CHANGES_REQUESTED** [review]

File: src/components/layout/Layout.tsx
Comment (ErikssonJoakim) [reply src/components/layout/Layout.tsx:203]: It was me who added this feature and I didn't want to use the index as it's not recommended, so I choose to use uuid to generate a unique key.

Comment (fredericvilcot) [reply src/components/layout/Layout.tsx:203]: Random keys are worst than indexes about performance and consistency.
Indexes are deprecated because they are weak (element could be moved or removed).
Keys must be under control, so please remove random uuids and use values instead (url or concatenation of values).

Comment (ErikssonJoakim) [reply src/components/layout/Layout.tsx:203]: Oki I was not aware of this, thanks for explaining 🙂  I will use the `namespace` instead as the url could be undefined which if I'm not mistaken would cause react to default to using index.

Comment (fredericvilcot) [reply src/components/layout/Layout.tsx:203]: @ErikssonJoakim 
From the React docs:

```
Keys should be stable, predictable, and unique. Unstable keys (like those produced by Math.random()) will cause many component instances and DOM nodes to be unnecessarily recreated, which can cause performance degradation and lost state in child components.
```

Comment (ccamel) [reply src/components/layout/Layout.tsx]: > Oki I was not aware of this, thanks for explaining 🙂  I will use the `namespace` instead as the url could be undefined which if I'm not mistaken would cause react to default to using index.

I'd just use the index since the list is constant.

**Review by JoeyDjr — APPROVED** [review]
LGTM

**Review by ccamel — APPROVED** [review]

**Review by fredericvilcot — APPROVED** [review]

**Pull request merged**
